vamb fails to install with python 3.11

Hi vamb team, I just tried to install vamb in a virtualenv with python 3.11 and it failed as follows:

Building wheels for collected packages: vamb
  Building wheel for vamb (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [66 lines of output]
      /bio/bin/python-env/lib/python3.11/site-packages/setuptools/dist.py:755: SetuptoolsDeprecationWarning: Invalid dash-separated options
      !!
      
              ********************************************************************************
              Usage of dash-separated 'description-file' will not be supported in future
              versions. Please use the underscore name 'description_file' instead.
      
              By 2023-Sep-26, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
      
              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************
      
      !!
        opt = self.warn_dash_deprecation(opt, section)
      /bio/bin/python-env/lib/python3.11/site-packages/setuptools/__init__.py:84: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
      !!
      
              ********************************************************************************
              Requirements should be satisfied by a PEP 517 installer.
              If you are using pip, you can try `pip install --use-pep517`.
              ********************************************************************************
      
      !!
        dist.fetch_build_eggs(dist.setup_requires)
      /bio/bin/python-env/lib/python3.11/site-packages/setuptools/dist.py:755: SetuptoolsDeprecationWarning: Invalid dash-separated options
      !!
      
              ********************************************************************************
              Usage of dash-separated 'description-file' will not be supported in future
              versions. Please use the underscore name 'description_file' instead.
      
              By 2023-Sep-26, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
      
              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************
      
      !!
        opt = self.warn_dash_deprecation(opt, section)
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-311
      creating build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/vambtools.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/parsecontigs.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/parsebam.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/encode.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/cluster.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/benchmark.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/__main__.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/__init__.py -> build/lib.linux-x86_64-cpython-311/vamb
      copying vamb/kernel.npz -> build/lib.linux-x86_64-cpython-311/vamb
      running build_ext
      skipping 'src/_vambtools.c' Cython extension (up-to-date)
      building 'vamb._vambtools' extension
      creating build/temp.linux-x86_64-cpython-311
      creating build/temp.linux-x86_64-cpython-311/src
      gcc -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -g -ffile-prefix-map=/build/python/src=/usr/src/debug/python -flto=auto -ffat-lto-objects -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -g -ffile-prefix-map=/build/python/src=/usr/src/debug/python -flto=auto -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -g -ffile-prefix-map=/build/python/src=/usr/src/debug/python -flto=auto -fPIC -I/bio/bin/python-env/include -I/usr/include/python3.11 -c src/_vambtools.c -o build/temp.linux-x86_64-cpython-311/src/_vambtools.o
      src/_vambtools.c:275:12: fatal error: longintrepr.h: No such file or directory
        275 |   #include "longintrepr.h"
            |            ^~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for vamb

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,7 +26,7 @@ version = {attr = "vamb.__version__"}
 readme = {file = "README.md"}
 
 [build-system]
-requires = ["setuptools ~= 63.0", "Cython ~= 0.29"]
+requires = ["setuptools ~= 63.0", "Cython ~= 0.29.5"]
 build-backend = "setuptools.build_meta"
 
 [tool.ruff]

